### ISSUE TITLE
browserstack provider is not seen when installed with yarn

### ISSUE THREAD

**Issue by nabrahamson** [DevExpress/testcafe-browser-provider-browserstack#50]
I am developing a project that is using yarn for our dependencies. When installing testcafe and testcafe-browser-provider-browserstack using the following commands it doesn't recognize the browserstack provider. 

`yarn add -D testcafe testcafe-browser-provider-browserstack`

I then run the command:

`testcafe -b browserstack`

I get the error message

`The specified "browserstack" browser provider was not found.`

If I install both locally with npm, then I get the expected results.

**Comment by AndreyBelym** [comment]
Hi @nabrahamson, have you seen the 'Using locally installed version of TestCafe' message when you start TestCafe? Could you please capture your directory structure by executing the `ls -R | grep node_modules >structure.txt` or `dir /s /b /a:d | findstr node_modules >structure.txt` from the root directory, and send it to me? You can also try to delete the `node_modules` directory in your project's root, and from all parent directories of your project's directory, and then try to reinstall modules by running `yarn install`.

**Comment by nabrahamson** [comment]
@AndreyBelym I'm not sure what has changed for me between yesterday and today. I had tried removing the node_modules directory and running the yarn install command and it had failed with the 'Using locally installed version of TestCafe' message. This morning after reading your message, I followed your steps in my project directory:

`rm -Rf node_modules`

`yarn install`

`testcafe -b browserstack`

The last command resulted in seeing the list of browsers instead of the error message above. Thank you for taking your time to look into this, but I must have just missed something trivial when trying to troubleshoot yesterday as your module is working fine with yarn now.
